docs/advanced-usage/deep-sleep.md

```diff
@@ -1,13 +1,20 @@
-Before deep sleeping, you will want to ensure that all messages are sent, including the `$online →  false`. To do that, you can call `Homie.prepareToSleep()`. This will disconnect everything cleanly, so that you can call `ESP.deepSleep()`.
+Before deep sleeping, you will want to ensure that all messages are sent, including property values of your `HomieNode` and the `$online →  false`. To do that, you can call `Homie.prepareToSleep()`. But the event `MQTT_READY` is emitted before your `HomieNode`s `loop()` method is called. Therefore you need to postpone the call to `Homie.prepareToSleep()` until the `loop()` method of all `HomieNode`s is called and their properties are submitted over MQTT. For that the additional Timer is necessary. Then `Homie.prepareToSleep()` will disconnect everything cleanly, so that you can call Homie.doDeepSleep()`.
```

Comment:
I think you missed a  \` here: Homie.doDeepSleep()\`

Reply:
Sometimes it is too late in the evening... Fixed.